packages/codemods/README.md

```diff
@@ -0,0 +1,83 @@
+# ts-morphin-migration
+
+ts-morphin-migration is a package created and managed with the `create-just` utility.
```

Comment:
Can you update this with the current package name and description? (remove the reference to create-just unless you did actually use that, and add a real description)